src/main/java/no/oslomet/aaas/model/AnonymizationPayload.java

```diff
@@ -1,21 +1,40 @@
 package no.oslomet.aaas.model;
 
+/***
+ *Model class for AnonymizationPayload.
```

Comment:
A bit redundant information here, should rather explain the usecase of the class